variables.tf

```diff
@@ -114,5 +120,21 @@ variable "tunneling_enabled" {
 variable "virtual_network_id" {
   type        = string
   default     = null
-  description = "The ID of the virtual network where the Azure Bastion Host is deployed."
+  description = "The ID of the virtual the Developer SKU Bastion hosts is attached to. Required for the Developer SKU Only."
+
+  validation {
+    condition     = (var.sku == "Developer" && var.virtual_network_id != null) || var.sku != "Developer" && var.virtual_network_id == null
+    error_message = "The virtual network ID is required for the Developer SKU (Only)."
+  }
+}
+
+variable "zones" {
+  type        = set(string)
+  default     = [1, 2, 3]

```

Comment:
The default is `set(number)`. Sorry to be a pedant, but would be better to use the same type as the variable.

Reply:
```suggestion
  default     = ["1", "2", "3"]

```

Reply:
No worries, i missed that. Thanks for fixing it